app/(authenticated)/employee-list/page.tsx

```diff
@@ -7,81 +7,92 @@ import plusImage from '@/assets/images/leadslist-icons/add-circle.png';
 import profileImage from '@/assets/images/profilePic.png';
 import { PAGE_ROUTES } from '@/utils/constants/common-constants';
 import EmployeelistRow from '@/components/row/employee-list-row';
+import { CREATE_EMPLOYEE_FORM_ITEMS } from '@/utils/constants/common-constants';
 import { useState } from 'react';
-import { EMPLOYEE_LIST_DATA_TYPE } from '@/models/global-types';
+import { EMPLOYEE_LIST_DATA_TYPE, CreateEmployeeItems } from '@/models/global-types';
```

Comment:
I believe we should rename the `EMPLOYEE_LIST_DATA_TYPE` data type to `EmployeeType`, that will be more convenient.